hoomd/md/pair.py

```diff
@@ -308,7 +320,7 @@ class SLJ(_Pair):
     R""" Shifted Lennard-Jones pair potential.
 
     Args:
-        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
+        nlist (:py:mod:`hoomd.md.nlist._Nlist`): Neighbor list
```

Comment:
```suggestion
        nlist (:py:mod:`hoomd.md.nlist._NList`): Neighbor list
```
This specific error is why the documentation test is not passing. It is in many places in this file, please find and fix them all.